torchvision/ops/boxes.py

```diff
@@ -325,13 +325,13 @@ def complete_box_iou(boxes1: Tensor, boxes2: Tensor, eps: float = 1e-7) -> Tenso
 
     diou, iou = _box_diou_iou(boxes1, boxes2, eps)
 
-    w_pred = boxes1[:, 2] - boxes1[:, 0]
-    h_pred = boxes1[:, 3] - boxes1[:, 1]
+    w_pred = boxes1[:, None, 2] - boxes1[:, None, 0]
```

Comment:
why we need this None for the pred and not for the gt boxes? I might be missing something obvious..

Reply:
Good question! I had the same broadcast originally but @fmassa pointed out to me that it can be simplified to this. See commit: https://github.com/pytorch/vision/pull/6338/commits/7bef3d047e33572bb84343f507401560be1bdbe9

Reply:
Thank you for the answer here https://github.com/pytorch/vision/pull/6338#discussion_r935351475